packages/itinerary-body/src/ItineraryBody/place-row.tsx

```diff
@@ -69,6 +69,11 @@ export default function PlaceRow({
     description: "Text describing the view-on-map button",
     id: "otpUi.ItineraryBody.viewOnMap"
   });
+
+  const formattedPlace = direction => (
+    <PlaceName config={config} interline={interline} place={direction} />
```

Comment:
(Note to create a GH issue) At one point, we replaced the interlined itinerary mock with one that is a different kind of interline, so this change is not illustrated in Storybook.